source/rf_wrapper.cpp

```diff
@@ -0,0 +1,20 @@
+#include "rf_wrapper.h"
+
+// Must be defined for next preprocessor tests to work
+#define ATMEL       0
+#define MCR20       1
+#define NCS36510    2
+
+#if MBED_CONF_APP_RADIO_TYPE == ATMEL
+#include "NanostackRfPhyAtmel.h"
+NanostackRfPhyAtmel rf_phy(ATMEL_SPI_MOSI, ATMEL_SPI_MISO, ATMEL_SPI_SCLK, ATMEL_SPI_CS,
```

Comment:
Can we enhance the code to check for Radio type too ? At least in case of ATMEL ? Just in case if someone wants to use SubG